src/seedu/addressbook/AddressBook.java

```diff
@@ -602,11 +609,21 @@ private static String getUserInput() {
         System.out.print(LINE_PREFIX + "Enter command: ");
         String inputLine = SCANNER.nextLine();
         // silently consume all blank and comment lines
-        while (inputLine.trim().isEmpty() || inputLine.trim().charAt(0) == INPUT_COMMENT_MARKER) {
-            inputLine = SCANNER.nextLine();
-        }
+        inputLine = consumeBlankAndCommentLines(inputLine);
         return inputLine;
     }
+    
+    /**
+     * Remove blank lines and comment lines from the text entered by the user
+     * @param inputLine
+     * @return inputLine without blank lines or comment lines
+     */
+	private static String consumeBlankAndCommentLines(String inputLine) {
```

Comment:
Good job for adding header comments. It should be `Removes` instead of `Remove` as you read it like `This method removes ...`. Also you may give some description for `inputLine`.